### ISSUE TITLE
Complete DMR

### ISSUE THREAD

**Issue by alwinhb** [USA-RedDragon/dmrgo#5]
@USA-RedDragon  Does this repo contains complete DMR Protocol [Data Link and Control] Implementation?

**Comment by USA-RedDragon** [comment]
No, this is much more of an untested "playground" for working with DMR in Go and is far from complete. <https://github.com/pd0mz/go-dmr> might have it, or <https://github.com/OK-DMR/ok-dmrlib> (python) and <https://github.com/rick51231/node-dmr-lib> (node) if you're not tied to Go
